src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -245,6 +263,9 @@ doclet.help.deprecated.body=\
 doclet.help.preview.body=\
     The {0} page lists all of the Preview APIs. \
     Preview APIs may be removed in future implementations.
+doclet.help.new.body=\
+    The {0} page lists APIs that have been added in recent releases. \
+    The content of this page is based on information provided by Javadoc @since tags.
```

Comment:
I removed the second sentence. 

There is no interaction with `-nosince`. I one point I thought `-nosince` should disable this feature, but actually both features are just different ways of displaying the information stored in `@since` tags that don't depend on each other.